front-end-components/src/views/compare-your-workforce/partials/total-teachers.tsx

```diff
@@ -68,6 +71,13 @@ export const TotalTeachers: React.FC<{ type: string; id: string }> = ({
         <h2 className="govuk-heading-m">
           Total number of teachers (Full Time Equivalent)
         </h2>
+        {hasIncompleteData ? (
```

Comment:
Why not have this logic in the component, and then pass hasIncompleteData as a prop?

Reply:
Immediately following the heading seems closer to the design of the other pages (warning near the top of the page) so being part of the children made sense but I don't think it would be a problem being displayed prior to the data instead. Happy to get this changed if needed.

Reply:
Think you've misunderstood. I'm suggested the below, if the conditional block in the component rather than being repeated in lots of places.

```       
<WarningBanner
            isVisible={hasIncompleteData }
            icon="!"
            visuallyHiddenText="Warning"
            message="Some schools don't have a complete set of financial data for this period"
/>
```

Reply:
Yes I completely misunderstood... That makes sense. I'll update in a new commit